<ui version="4.0" >
 <class>HistoryPanel</class>
 <widget class="QWidget" name="HistoryPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>History</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="spacing" >
    <number>3</number>
   </property>
   <property name="margin" >
    <number>1</number>
   </property>
   <item>
    <widget class="QListWidget" name="listWidget" />
   </item>
  </layout>
  <action name="ui_undoAction" >
   <property name="text" >
    <string>Undo</string>
   </property>
   <property name="icon" >
     <iconset resource="../resources/gui.qrc" >
       <normaloff>:/undoIcon</normaloff>:/undoIcon
     </iconset>
   </property>
 </action>
 <action name="ui_redoAction" >
   <property name="text" >
     <string>Redo</string>
   </property>
   <property name="icon" >
     <iconset resource="../resources/gui.qrc" >
     <normaloff>:/redoIcon</normaloff>:/redoIcon</iconset>
   </property>
  </action>
  <action name="ui_clearAction" >
   <property name="text" >
    <string>Clear</string>
   </property>
   <property name="icon" >
     <iconset resource="../resources/gui.qrc" >
     <normaloff>:/clearHistoryIcon</normaloff>:/clearHistoryIcon</iconset>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
